### PR TITLE
[corlib] Link out more SR/SRE types

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

File: mcs/class/corlib/System.Reflection.Emit/TypeBuilder.cs [hunk 1]
```diff
@@ -120,8 +120,15 @@ protected override TypeAttributes GetAttributeFlagsImpl ()
 			return attrs;
 		}
 
+		TypeBuilder ()
```
Comment (marek-safar) [inline mcs/class/corlib/System.Reflection.Emit/TypeBuilder.cs]: This will have to be manually ported to dotnet/runtime

Comment (vargaz) [reply mcs/class/corlib/System.Reflection.Emit/TypeBuilder.cs]: This is just for normal mono, netcore mono has different code.

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
Removed some changes which might be affected by https://github.com/mono/linker/issues/965

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
netcore is more complicated.

**Pull request merged**